git-branchless/src/opts.rs

```diff
@@ -399,8 +399,6 @@ pub enum Command {
     /// Reword commits.
     Reword {
         /// Zero or more commits to reword. If not provided, defaults to "HEAD".
-        ///
-        /// Can either be hashes, like `abc123`, or ref-specs, like `HEAD^`.
```

Comment:
This is related, but I removed this b/c we don't include a similar explanation in, eg, `git move`, and I also think it's outdated. (eg, it doesn't mention revsets)